src/Grpc.AspNetCore.Server/Internal/GrpcProtocolConstants.cs

```diff
@@ -20,6 +20,7 @@ namespace Grpc.AspNetCore.Server.Internal
 {
     internal static class GrpcProtocolConstants
     {
+        internal const string TimeoutHeader = "grpc-timeout";
```

Comment:
Something we should probably address at some point:
Not sure if we are excluding these headers from metadata exposed via ServerCallContext, but in general the grpc-specific headers that have special handling in the framework shouldn't be exposed by visible in metadata (Grpc.Core doesn't expose them because the c-core also doesn't expose them).